cmd/soci/commands/ztoc/get-file.go

```diff
@@ -25,6 +25,8 @@ import (
 
 	"github.com/awslabs/soci-snapshotter/fs/config"
 	"github.com/awslabs/soci-snapshotter/soci"
+	"github.com/awslabs/soci-snapshotter/ztoc"
+	ztc "github.com/awslabs/soci-snapshotter/ztoc"
```

Comment:
How about renaming the variable `ztoc` to something else (toc?) rather than renaming the package to ztc here?

Reply:
That's a great idea.

Reply:
Done.

Reply:
Done.